openedx_external_enrollments/external_enrollments/base_external_enrollment.py

```diff
@@ -56,13 +56,27 @@ def _post_enrollment(self, data, course_settings=None):
             )
             return str(error), status.HTTP_400_BAD_REQUEST
         else:
-            LOG.info('External enrollment response for [%s] -- %s', self.__str__(), response.json())
-            log_details["response"] = response.json()
+            json_response = self._get_json_response(response)
+
+            LOG.info('External enrollment response for [%s] -- %s', self.__str__(), json_response)
+            log_details["response"] = json_response
```

Comment:
Please do not mix double and single quotes, just one of them. @ivanvgh

Reply:
Changed to double-quotes throughout the whole file.